uni/lib/view/common_widgets/toast_message.dart

```diff
@@ -1,17 +1,122 @@
 import 'package:flutter/material.dart';
-import 'package:toast/toast.dart';
+import 'package:flutter/cupertino.dart';
 
 /// Provides feedback about an operation in a small popup
 ///
 /// usage example: ToastMessage.display(context, toastMsg);
+class MessageToast extends StatelessWidget {
+  final String message;
+  final Color? color;
+  final IconData? icon;
+  final Color? iconColor;
+  final Color? textColor;
+  final AlignmentGeometry? alignment;
+  final dynamic elevation;
+
+  const MessageToast(
+      {Key? key,
+      required this.message,
+      this.color = Colors.white,
+      required this.icon,
+      this.iconColor = Colors.black,
+      this.textColor = Colors.black,
+      this.alignment = Alignment.bottomCenter,
+      this.elevation = 0.0})
```

Comment:
Tip: Format code with CTRL-ALT-L

Reply:
It says it's already formated.

<img width="633" alt="image" src="https://user-images.githubusercontent.com/87575749/202784028-8eeacdc2-4047-4073-a55a-7e5cb63d2d91.png">

Reply:
My bad, My IDE somehow had changed the "{Key? key" to line 16